src/Pages/NavigatorDashboard.js

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useState } from "react";
+import { collection, query, where, getDocs, doc, update } from "firebase/firestore";
+import { db } from "../Components/firebase";
+import "./NavigatorDashboard.css"
+import Checkboxes from "./Checkboxes";
+
+// later we can make each tab a different component, the individual tabs take a jobseeker as a prob
+
+
+
+
+function NavigatorDashboard() {
+
+    const updateJobseeker = async () => {
+        const ref = doc(db, 'jobseekers', {seekerID})
+        //await update need to update db first
+    };
+
+    const [seekerID, setSeekerID] = useState('');
+    const [jobseeker, setJobseeker] = useState({
+        name: "Name",
+        pronouns: "Pronouns",
+        phone: "Phone",
+        email: "Email",
+        ethnicity: "Ethnicity",
+        degree: "No",
+        degreeType: "None",
+        certificate: "No",
+        certificateType: "None",
+    })
+    const {name, pronouns, phone, email, ethnicity, degree, degreeType, certificate, certificateType} = jobseeker
+
+    useEffect(() => {
+        getSeeker(setJobseeker)
+    }, [setJobseeker])
+    const skills = ['Accounting Software', 'Administrative', 'Adobe Software Suite', 'Bilingual', 'Brand Management', 'Cold Calling'] //can expand to all skills on spreadsheet
+    
+    const [checkedArr, setCheckedArr] = useState(new Array(skills.length).fill(false))
+    let props = {skills, checkedArr, setCheckedArr}
+
+
+    const getSeeker = async (setJobseeker) => {
+        const Ref = collection(db, "jobseekers");
+    
+    const q = query(Ref, where("name", "==", "Sol I Adams"));
+    const querySnapshot = await getDocs(q);
+    querySnapshot.forEach((doc) => {
+        const data = doc.data();
+        setSeekerID(doc.id)
+        setJobseeker({
+            name: data.name,
+            pronouns: data.pronouns,
+            phone: data.phone,
+            email: data.email,
+            ethnicity: data.ethnicity,
+            degree: data.degree,
+            degreeType: data.degreeType,
+            certificate: data.certificate,
+            certificateType: data.certificateType,
+        })
+    });
+    }
+
+  return (
+    <div>
+        <div>
+            [Image goes here]
```

Comment:
can you add an avatar//img component w a default value? When the designers are done with the profile pictures we will pass this in as an additional attr of the jobseeker prop